main/src/com/google/refine/model/recon/StandardReconConfig.java

```diff
@@ -217,12 +223,26 @@ public ReconType getReconType() {
         }
         return null;
     }
+        @Override
+        @JsonIgnore
+        public int getBatchSize (Project project) {
+        if((project.rows.size()/100)>10)
+        {
+            if((project.rows.size()/100)<batchSize)
+            {
+                return (project.rows.size()/100);
+            }
+            else
+            {
+                if(batchSize!=0)
+                {
+                    return batchSize;
+                }
+            }
```

Comment:
I think judicious use of `Math.max()` & `Math.min()` might make this more concise and easier to understand. Something along the lines of this (untested) code:
```suggestion
        return Math.min(Math.max(row_count/100, 10), manifest_batch_size);
```